plugins/linear-genome-view/src/LinearGenomeView/components/Header.tsx

```diff
@@ -173,6 +184,7 @@ const LinearGenomeViewHeader = observer(
           <RefNameAutocomplete
             onSelect={handleSelectedRegion}
             assemblyName={assemblyName}
+            fetchResults={fetchResults}
```

Comment:
We moved the fetch results out of the refNameAutocomplete? Is this for more usability in other components?

Reply:
yes. this was a architetural idea. it does allow for a little more usability outside lgv this way, or "inversion of control" so the place where it is used can dictate it's how it behaves.

Reply:
okie! good call :) 👍